encrypted-media-respec.html

```diff
@@ -3969,33 +3969,29 @@
         </section>
         <section>
           <h4>Mitigations</h4>
-          <p>In summary, user agent and CDM implementations that allow the CDM to persist data:</p>
-          <ul>
-            <li><p>MUST ensure it is restricted to the <a def-id="origin"></a> for which it was created.</p></li>
-            <li><p>MUST ensure it is restricted to the current <a def-id="browsing-profile"></a> and does not leak to or from private browsing sessions.</p></li>
-            <li><p>MUST allow the user to clear it.</p></li>
-            <li><p>SHOULD allow the user to clear it by <a def-id="origin"></a>.</p></li>
-            <li><p>SHOULD treat it like other site data, including presenting it along with cookies [[!COOKIES]], including it in "remove all data", and presenting it in the same UI locations.</p></li>
-          </ul>
-
           <dl>
-            <dt>Origin-specific Key System storage</dt>
+            <dt>Use Origin-specific and user agent profile-specific Key System storage</dt>
```

Comment:
"Use" is inconsistent with other headers in this section but consistent with those in other sections. I think this is probably better. I'll make them all consistent after this lands.